examples/rio_stac_task.py

```diff
@@ -0,0 +1,12 @@
+import rio_stac.stac
+import stac_task
+from pystac import Item
+from stac_task import HrefTask
+
+
+class RioStacTask(HrefTask):
```

Comment:
An example here on how I think this should be based on my other comments here (although here I'm using args and kwargs - required keywords are maybe better instead of args?)

```
class RioStacTask(StacOutputTask):
    def process(self,  href: str):
        return rio_stac.stac.create_stac_item(href)
```

Reply:
See https://github.com/stac-utils/stac-task/pull/42#discussion_r1330322311, I think task parameters should be defined at task instantiation, not at process-time.

Reply:
I see now how the parameters are defined at instantiation and become class members. I think that's fine, just a change from the older paradigm I'm used to.